qds_sdk/cluster.py

```diff
@@ -216,6 +216,13 @@ def _parse_create_update(cls, args, action):
                              default=None,
                              help="Use hadoop1 instead of hadoop2. This is the default.")
 
+        spark = hadoop_group.add_mutually_exclusive_group()
+        spark.add_argument("--use-spark",
+                           dest="use_spark",
+                           action="store_true",
+                           default=None,
+                           help="Turn on spark for this cluster")
```

Comment:
I have my doubts. This is required for testing. However in production, what does it mean to make it a non-spark cluster or non-hbase cluster ? The affects can be devastating. Should we make it difficult to change type of clusters ? 
Users can always create new clusters and re-label them properly.